src/tests/public_api/test_helpers/proxy.h

```diff
@@ -119,9 +121,129 @@ class Proxy : public core::Thread, private packet::IWriter {
               == 0);
     }
 
+    Proxy(const roc_endpoint* receiver_source_endp,
+          const roc_endpoint* receiver_repair_endp,
+          const roc_endpoint* receiver_control_endp,
+          size_t n_source_packets,
+          size_t n_repair_packets,
+          size_t n_control_packets,
```

Comment:
`n_control_packets` is not a thing. source and repair packets are grouped into a FEC block, but control packets are not part of any block. We can just remove this parameter.